### ISSUE TITLE
[Bug]  The Swipe view does not call the command associated to it when tapped

### ISSUE THREAD

**Issue by DamienDoumer** [xamarin/Xamarin.Forms#9028]
### Description
When an item is added to the swipe view, and a command is added to the swipe view, the swipeview won't run that command when its item is tapped. 

### Steps to Reproduce

1. Add a swipe view to an item template in xaml
2. Add an item to the swipe view and a command to it, let the swipe view be in default mode 
3. Run the app, reveal the swipe view items and tap the item, 

### Expected Behavior
The swipe view should run the command which corresponds to the swipe item tapped.

### Actual Behavior
After revealing the item, if it is tapped, the swipe view only hides the item after it is tapped and does not run the command associated to it.

### Basic Information

- Version with issue: Xamarin Forms 4.4.099
- IDE: Visualstudio Enterprise 2019
- Platform Target Frameworks:  android
  - Android: 9, Pie
- Affected Devices: Honor 9 Lite

### Screenshots

![xforms swipeview error](https://user-images.githubusercontent.com/16731017/71520566-15545600-28bd-11ea-9589-8cb1caffeaa7.gif)

### Reproduction Link

I made a demo project with reproduction of this bug.  The project is the same as what you see in teh gif above. You can find it here: https://github.com/DamienDoumer/XFormsSwipeViewBugRepro

**Comment by haavamoa** [comment]
Can this be the same bug that you experoenced @ranwala?

**Comment by ranwala** [comment]
yes, this is the same bug currently i am experienced with, this is working with simulators but not in any device.

**Comment by neville-nazerane** [comment]
It's not just the swipe items. No click event inside a `SwipeView` seems to be working on android. Even anything in its contents. 

I tried a grid in the content with tap gester recognizer. It worked fine without the SwipeView but not with. I even tried one of the components inside the grid having a tap gesture recognizer. It still doesn't work. 

The Invoked event seems to work on emulators but not devices. All commands work on neither. 

Details:
Visual Studio Community Version 16.4.2
Xamarin Forms 4.4.0.991265
Emulator: Pixel 3

**Comment by chrisparkeronline** [comment]
I am having the same issue inside of a collectionview.

**Comment by vhugogarcia** [comment]
I am facing the exact same issue posted on this issue. For me it is happening on Android and iOS, simulators and real devices, both of them.

**Comment by NNebus** [comment]
Having the exact same issue on Android 9.0 (Huawei Mate 20 Lite - SNE-LX1).
Got another real device with Android 7.0 (Huawei P9 Lite - VNS-L31) but on this device everything is working fine.

Microsoft Visual Studio Community 2019 Version 16.4.2
Xamarin.Forms Version 4.5.0.142-pre1

Even the [SwipeViewDemos ](https://github.com/xamarin/xamarin-forms-samples/tree/master/UserInterface/SwipeViewDemos)have those issue so it's not an implementation failure.

**Comment by ianvink** [comment]
I can confirm I get this too on all devices, iOS, and Android. SwipeView stops the tap of the row in a CollectionView/ListView

**Comment by kaminipatel03** [comment]
Anyone found the solution of tap event in SwipView?
I am still having the problem when click on SwipItem.  I have updated xamarian.forms to 4.5.0.187-pre2 but still having same problem.

Thanks in advance

**Comment by NNebus** [comment]
@kaminipatel03 please notice the PR for this issue: https://github.com/xamarin/Xamarin.Forms/pull/9177

**Comment by neville-nazerane** [comment]
The issue still persists in the latest nightly build. I am testing it with a Note 10 as of now.

**Comment by kienczkowski** [comment]
Hi, 

I have Huawei P30 Lite. I had this some issue early. In version Xamarin Froms 4.5.0.395 this issue is didn't occures. In my case I have SwipeView in CollectionView, it's works great. Thank you for your support !!! ;)

**Comment by luczinski** [comment]
Hi guys! 
I'm experiencing this bug with Xamarin.Forms 5.0.0.2012 testing on Android. What’s the status of this bug?
Thanks!

**Comment by jpmirfcb** [comment]
> Hi guys!
> I'm experiencing this bug with Xamarin.Forms 5.0.0.2012 testing on Android. What’s the status of this bug?
> Thanks!

Same here.

**Comment by seanLindholm** [comment]
For anyone still having trouble with this a workout can be to self bindings to access any command specified in this context.
As an example I build a collectionview with three swipitems (1 left and 2 right) using a MVVM architecture. 
Using the relative ancestor binding, only the left one worked, but using self binding, commands can now be activated for all items.

Referring to the documentation on binding https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/data-binding/relative-bindings make sure that type is consistent.

I might not solve all problems, but it solved mine. 

TLDR;

Changed command expression from: 

Command="{Binding Source={RelativeSource AncestorType={x:Type vm:ItemDetailViewModel}}, Path=EditItemCommand}"

To

Command="{Binding Source={x:Reference collectionView}, Path=BindingContext.EditItemCommand}"

**Comment by maxalmonte14** [comment]
@seanLindholm your solution worked for me, I don't know why it wouldn't use the same binding context as the collection view by default tho, hope they solve this.
